app/components/post_list/post/body/message/message.tsx

```diff
@@ -61,9 +63,13 @@ const Message = ({currentUser, groupsForPosts, highlight, isEdited, isPendingOrF
     const style = getStyleSheet(theme);
     const blockStyles = getMarkdownBlockStyles(theme);
     const textStyles = getMarkdownTextStyles(theme);
+
+    // groups: MM-41882 const mentionKeys = useMemo(() => {
+    // groups: MM-41882     return getMentionKeysForPost(currentUser, post, groupsForPosts);
+    // groups: MM-41882 }, [currentUser, post.message, groupsForPosts]);
     const mentionKeys = useMemo(() => {
-        return getMentionKeysForPost(currentUser, post, groupsForPosts);
-    }, [currentUser, post.message, groupsForPosts]);
+        return getMentionKeysForPost(currentUser);
```

Comment:
```suggestion
        return currentUser.mentionKeys;
```

Reply:
I'm thinking that we should also remove the function `getMentionKeysForPost`

Reply:
also done.